README.md

```diff
@@ -100,7 +100,53 @@ public String TrimString(String s) {
 ## Class members
 
 - Avoid `this.` unless absolutely necessary.
-- Always specify member visiblity, even if it's the default (i.e. `private string foo;` not `string _foo;`).
+
+DO
+```
+public class Node 
```

Comment:
Both are the clear examples of how to use `this`. Let's use your code to demonstrate the approach - I don't mind.